src/js/with-react.js

```diff
@@ -108,6 +112,14 @@ const withReact = (WebComponent, { pure = true, passive = false } = {}) => {
       const [eventKeys, dataKeys] = propsKeys.reduce(partition(isEventFilter), [[], []]);
 
       eventKeys.forEach((key) => {
+        // supper React's synthetic events
+        // Note: not 100% compatible with custom elements
+        if (!isNamespacedEvent(key)) {
```

Comment:
@MarekLacoAXA 
@LucaMele 
To support React's synthetic events we would need to implement some special handling:
https://reactjs.org/docs/web-components.html